core/client/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -161,6 +161,24 @@ public void hadoopShouldLoadFileSystemWhenConfiguredTest() throws Exception {
   }
 
   /**
+  * Ensures that when ListStatusNonExisting should Throw FileNotFoundException.
+  */
+  @Test
+  public void shouldThrowFileNotFoundExceptionWhenListStatusNonExisting() throws Exception {
+    StringBuilder path = new StringBuilder("/ALLUXIO-2036.");
+    path.append(System.currentTimeMillis()).append(".txt");
+    FileSystem fs = new FileSystem();
+    try {
+      fs.listStatus(new Path(path.toString()));
+      Assert.fail("Listing the status of a nonexistent file should throw an exception");
+    } catch (FileNotFoundException e) {
+      LOG.info(e.getMessage());
```

Comment:
This can just be `// This exception is expected` instead of logging. Logging could make it look like there was an error even when the test passes.